app/models/comment.rb

```diff
@@ -327,13 +321,14 @@ def async_bust
   handle_asynchronously :async_bust
 
   def send_email_notification
-    NotifyMailer.new_reply_email(self).deliver if parent_email_exist?
+    NotifyMailer.new_reply_email(self).deliver
```

Comment:
`send_email_notification` will only be queued up to delayed_job if `should_send_email_notification?` is true.